exonum/src/node/state.rs

```diff
@@ -1126,4 +1127,20 @@ impl State {
     pub fn set_our_connect_message(&mut self, msg: Connect) {
         self.our_connect_message = msg;
     }
+
+    /// Add peer to node's `ConnectList`.
+    pub fn add_peer_to_connect_list(&mut self, peer: ConnectInfo) -> bool {
+        match self.find_validator(peer.public_key) {
```

Comment:
- `if _.is_some()` looks more natural
- More importantly, does this code simply ignore all auditors? Is this intended?